data/formats-data.js

```diff
@@ -3001,7 +3007,7 @@ exports.BattleFormatsData = {
 	},
 	flygon: {
 		randomBattleMoves: ["earthquake", "outrage", "uturn", "roost", "stoneedge", "firepunch", "dragondance"],
-		randomDoubleBattleMoves: ["earthquake", "protect", "dragonclaw", "uturn", "rockslide", "firepunch", "fireblast", "tailwind", "feint"],
+		randomDoubleBattleMoves: ["earthquake", "protect", "dragonclaw", "uturn", "rockslide", "firepunch", "fireblast", "tailwind", "feint", "dragondance"],
```

Comment:
Actually, Feint off a base 100 without STAB or an ability to boost it is not very great. Remove it here